### PR TITLE
PSR 415

### PR DESCRIPTION
**Pull request by EricSiegNW**
This adds this "Insights Complete" view. It shows instead of the insight unlocked view if there aren't anymore insights to share. Note: Looks fine on bigger phones, but will need a smaller amount of text for the touch.

### CODE REVIEW

**Comment by dephillipsmichael** [comment]
It's cut-off even on iPhone 12 which has a big screen.  Is the height hard-coded?  Should expand to fit the text.

Screen Shot 2021-06-25 at 11 05 19 AM

**Comment by dephillipsmichael** [comment]
I don't really understand the point of suggesting the user go check out their movies when we have pop-tips specifically for that scenario, and if the user completed their insights, they most likely already know about the review tab.

File: Psorcast/PsorcastValidation/Psorcast.strings [hunk 1]
```diff
@@ -117,6 +117,7 @@
 "INSIGHT_UNLOCKED_TEXT" = "View Your Insight";
 "INSIGHT_YES_BUTTON" = "Yes";
 "INSIGHT_NOT_REALLY_BUTTON" = "Not Really";
+"INSIGHTS_COMPLETE_TEXT" = "Thank you from all of us at Psorcast, we’ll see you next week and check out your movies after you have more than 1 measurement.";
```
Comment (DanWebster) [inline Psorcast/PsorcastValidation/Psorcast.strings:120]:
```suggestion
"INSIGHTS_COMPLETE_TEXT" = "Thank you from all of us at Psorcast, keep up the great work next week!";
```

**Review by DanWebster — CHANGES_REQUESTED** [review]
Put in a different text to present (can just commit here in github) upon completing activities for a given week

**Comment by DanWebster** [comment]
> I don't really understand the point of suggesting the user go check out their movies when we have pop-tips specifically for that scenario, and if the user completed their insights, they most likely already know about the review tab.

Good call, updated copy included in my review

**Pull request merged**